src/main/groovy/de/dkfz/roddy/tools/RoddyIOHelperMethods.groovy

```diff
@@ -162,16 +163,17 @@ class RoddyIOHelperMethods {
     }
 
     /**
-     * This method is a replacement for Apache FileUtils.copyDirectory
-     * It uses the local Roddy command set (with e.g. "cp -r" for Bash shells)
-     * It is designed to retain access rights.
+     * This method copies a file and tries to preserve access rights using Java get/set rights methods.
      * @param src
      * @param tgt
      */
-    public static void copyDirectory(String cmd) {
+    public static void copyDirectory(File src, File tgt) {
         try {
-            //String cmd = Roddy.getLocalCommandSet().getCopyDirectoryCommand(src, tgt);
-            ExecutionHelper.executeSingleCommand(cmd);
+            FileUtils.copyDirectory(src, tgt)
+            tgt.setReadable(src.canRead())
+            tgt.setWritable(src.canWrite())
+            tgt.setExecutable(src.canExecute())
+
         } catch (Exception ex) {
             logger.severe(ex.toString())
```

Comment:
Maybe we can add a switch to turn off (not on!) the exception. Thus the function can be used in both contexts -- strict (with exception) and non-strict.